### PR TITLE
Switch to binary classification for PU

### PR DESCRIPTION
**Pull request by kawaho**
This PR switches PU regression to binary classification, as the majority of particles are either isPU=1 or 0. Focal loss was tried but do not offer any benefits compared to cross entropy. The final output of the PU branch may be tuned for a more optimized threshold for PU identification.

### CODE REVIEW

File: parameters/pytorch/pyg-cms.yaml [hunk 1]
```diff
@@ -118,9 +118,6 @@ train_dataset:
         cms_pf_qcd:
           version: 2.5.0
           splits: [1,2,3,4,5,6,7,8,9,10]
-        cms_pf_ztt:
-          version: 2.5.0
-          splits: [1,2,3,4,5,6,7,8,9,10]
```
Comment (jpata) [inline parameters/pytorch/pyg-cms.yaml:123]: did you intend to remove these?

**Comment by jpata** [comment]
Thanks, looks good! Just had a minor question inline (see above): https://github.com/jpata/particleflow/pull/402#discussion_r1983050600

**Comment by kawaho** [comment]
I think it was simply because we didn't copy the ztt dataset. I reverted it. Thanks.
